cmake/linux/generate_icons_linux.sh.in

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# Recolor icon
+echo Recoloring vector icon...
+case "$1" in
+	"-r" ) # Release
+	sed 's/#780116/#249a56/g; s/#c51306/#50d99b/g' ../branding/icon.svg > recolored-icon.svg
+	sed 's/#780116/#249a56/g; s/#c51306/#50d99b/g' ../branding/splash.svg > recolored-splash.svg
+	;;
+	"-b" | "-rc" ) # Beta/Release Candidate
+	sed 's/#780116/#0267c1/g; s/#c51306/#09a9d9/g' ../branding/icon.svg > recolored-icon.svg
+	sed 's/#780116/#0267c1/g; s/#c51306/#09a9d9/g' ../branding/splash.svg > recolored-splash.svg
+	;;
+	"-c" | "-n" ) # Canary/Nightly
+	sed 's/#780116/#ffa40f/g; s/#c51306/#fad200/g' ../branding/icon.svg > recolored-icon.svg
+	sed 's/#780116/#ffa40f/g; s/#c51306/#fad200/g' ../branding/splash.svg > recolored-splash.svg
+	;;
+	* ) # No type specified, leave it be
+	sed '' ../branding/icon.svg > recolored-icon.svg
+	sed '' ../branding/splash.svg > recolored-splash.svg
+	;;
+esac
+
+# Generate raster images
+echo Generating raster images for icon...
```

Comment:
Please add double quotes to all calls to `echo`.  More information is here: https://unix.stackexchange.com/a/269232/190347.

Reply:
Fixed 👍